### DISCUSSION TITLE
no projects visible in docker container after sync is done

### DISCUSSION THREAD

**Discussion by dayDPTR** [oracle/opengrok#4436] [General]
**Describe the bug**
before run opengrok docker image, I created the folder: /root/opengrok/src/, and git clone below repositories from github as below:
```
root@M5L:/home/matrix# ls /root/opengrok/src/
click  hostap  openssl  openwrt  packages  radsecproxy
```
then I run opengrok docker image as below:
```
root@M5L:/home/matrix# docker run -d --name opengrok -p 8888:8080/tcp -e SYNC_PERIOD_MINUTES="10" -v ~/opengrok/src/:/opengrok/src/ -v ~/opengrok/etc/:/opengrok/etc/ -v ~/opengrok/data/:/opengrok/data/ opengrok/docker:1.12
6b4d80f13323daedb56a416f81fac6afcbed2e6a64051a83b6593ef7ed702ecb
```
opengrok docker image version: 1.12, shown as below:
```
CONTAINER ID   IMAGE                  COMMAND               CREATED          STATUS          PORTS                                       NAMES
6b4d80f13323   opengrok/docker:1.12   "/scripts/start.py"   14 minutes ago   Up 14 minutes   0.0.0.0:8888->8080/tcp, :::8888->8080/tcp   opengrok
```
However, when I accessed [localhost:8888](localhost:8888), there is no any project shown at webpage:
<img width="1119" alt="Screenshot 2023-10-16 at 16 09 57" src="https://github.com/oracle/opengrok/assets/13374339/a2629fc8-89f6-4e38-a0bf-9bd3879b0889">

docker version:
```
root@M5L:/home/matrix# docker version
Client: Docker Engine - Community
 Version:           24.0.2
 API version:       1.43
 Go version:        go1.20.4
 Git commit:        cb74dfc
 Built:             Thu May 25 21:52:13 2023
 OS/Arch:           linux/amd64
 Context:           default

Server: Docker Engine - Community
 Engine:
  Version:          24.0.2
  API version:      1.43 (minimum version 1.12)
  Go version:       go1.20.4
  Git commit:       659604f
  Built:            Thu May 25 21:52:13 2023
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.6.21
  GitCommit:        3dce8eb055cbb6872793272b4f20ed16117344f8
 runc:
  Version:          1.1.7
  GitCommit:        v1.1.7-0-g860f061
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

**To Reproduce**
1. execute above command to run opengrok docker images;
2. access localhost:8888 to check if projects at /opengrok/src are shown at the webpage.

**Expected behavior**
projects at /opengrok/src can be shown at the webpage.

**Screenshots**

**Additional context**

**Comment by vladak** [comment]
Seems like there is a problem with the projects not being marked as indexed.

**Reply by dayDPTR** [reply]
so how to fix it ?

**Reply by a4081675** [reply]
Has this problem ever been solved? I'm having the same problem.

**Reply by ydsood** [reply]
I am noticing the same issue with svn repositories
